src/core/Slider/Slider.test.tsx

```diff
@@ -0,0 +1,934 @@
+        type: any,
+        { pointerId = 1, pointerType = 'mouse', ...rest } = {},
+      ) {
+        const event = fn(type, rest);
+        event.pointerId = pointerId;
+        event.pointerType = pointerType;
+        const eventType = event.type;
+        Object.defineProperty(event, 'type', {
+          get: function () {
+            return eventType.replace('mouse', 'pointer');
+          },
+        });
+        return event;
+      };
+    }
+  }
+}
+
+patchCreateEvent(createEvent);
+
+const createBoundingClientRect = (
+  left: number,
+  top: number,
+  right: number,
+  bottom: number,
+): DOMRect => ({
+  left,
+  top,
+  right,
+  bottom,
+  width: right - left,
+  height: bottom - top,
+  x: left,
+  y: top,
+  toJSON: () => '',
+});
+
+/**
+ * Setup default size for slider container to be used by all slider tests
+ */
+const getBoundingClientRect = Element.prototype.getBoundingClientRect;
+const sliderContainerSize = createBoundingClientRect(10, 0, 1010, 60);
+Element.prototype.getBoundingClientRect = () => sliderContainerSize;
+
+afterAll(() => {
+  Element.prototype.getBoundingClientRect = getBoundingClientRect;
+});
+
+const defaultSingleValue = [50];
+
+const assertBaseElement = (container: HTMLElement) => {
+  expect(
+    container.querySelector('.iui-slider-component-container'),
+  ).toBeTruthy();
+};
+
+it('should render correctly in its most basic state', () => {
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('false');
+});
+it('should reset callbacks on value change', () => {
+  const wrapper = render(<Slider values={defaultSingleValue} />);
+  const { container } = wrapper;
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  wrapper.rerender(<Slider values={[10, 20]} max={90} />);
+});
+it('should not render thumbs if no values are defined', () => {
+  const { container } = render(<Slider values={[]} />);
+  assertBaseElement(container);
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeFalsy();
+});
+
+it('should render disabled component', () => {
+  const { container } = render(<Slider values={defaultSingleValue} disabled />);
+  assertBaseElement(container);
+  expect(
+    container.querySelector('.iui-slider-component-container.iui-disabled'),
+  ).toBeTruthy();
+  const thumb = container.querySelector('.iui-slider-thumb') as HTMLDivElement;
+  expect(thumb).toBeTruthy();
+  expect(thumb.getAttribute('aria-disabled')).toEqual('true');
+});
+it('should render min max labels by default', () => {
+  const { container } = render(<Slider values={defaultSingleValue} />);
+  assertBaseElement(container);
+  expect(
```

Comment:
These chekcs could have been a part of first (basic) test. We usually check all the things which are expected to be rendered in there (when you do not pass optional props). So apart from these rail with track could be checked there too.